ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/events/TransactionsReader.scala

```diff
@@ -97,14 +101,12 @@ private[appendonlydao] final class TransactionsReader(
     val query = (range: EventsRange[(Offset, Long)]) => {
       implicit connection: Connection =>
         logger.debug(s"getFlatTransactions query($range)")
-        QueryNonPruned.executeSqlOrThrow(
-          EventsTableFlatEvents
-            .preparePagedGetFlatTransactions(sqlFunctions)(
-              range = EventsRange(range.startExclusive._2, range.endInclusive._2),
-              filter = filter,
-              pageSize = pageSize,
-            )
-            .executeSql,
+        queryNonPruned.executeSqlOrThrow(
```

Comment:
The asymmetry between the two functions returning sources: `getFlatTransactions` and `getTransactionTrees` existed before, but maybe it is worth leaving a note that it should be unified.

Reply:
I left a TODO for that :ok_hand: